src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCardView/HomeStatisticsCardView.swift

```diff
@@ -46,47 +46,57 @@ class HomeStatisticsCardView: UIView {
 			var accessibilityElements = [Any]()
 
 			if viewModel?.title != nil, let titleLabel = self.titleLabel {
+				titleLabel.accessibilityTraits = UIAccessibilityTraits.header
 				accessibilityElements.append(titleLabel)
 			}
 
 			if let infoButton = self.infoButton {
 				accessibilityElements.append(infoButton)
+				infoButton.accessibilityTraits = UIAccessibilityTraits.button
 				infoButton.accessibilityIdentifier = viewModel?.infoButtonAccessiblityIdentifier
 			}
 
 			if viewModel?.primaryTitle != nil, let primaryTitleLabel = self.primaryTitleLabel {
+				var primaryAccessibilityLabel = primaryTitleLabel.text
+				if viewModel?.primaryValue != nil, let primaryValueLabel = self.primaryValueLabel {
+					primaryAccessibilityLabel?.append(" ")
+					primaryAccessibilityLabel?.append(primaryValueLabel.text ?? "")
+				}
+				
+				primaryTitleLabel.accessibilityLabel = primaryAccessibilityLabel
 				accessibilityElements.append(primaryTitleLabel)
 			}
 
-			if viewModel?.primaryValue != nil, let primaryValueLabel = self.primaryValueLabel {
-				accessibilityElements.append(primaryValueLabel)
-			}
-
 			if viewModel?.primaryTrendImage != nil, let primaryTrendImageView = self.primaryTrendImageView {
 				accessibilityElements.append(primaryTrendImageView)
 			}
 
 			if viewModel?.secondaryTitle != nil, let secondaryTitleLabel = self.secondaryTitleLabel {
+				var secondaryAccessibilityLabel = secondaryTitleLabel.text
+				if viewModel?.secondaryValue != nil, let secondaryValueLabel = self.secondaryValueLabel {
+					secondaryAccessibilityLabel?.append(" ")
+					secondaryAccessibilityLabel?.append(secondaryValueLabel.text ?? "")
+				}
+				secondaryTitleLabel.accessibilityLabel = secondaryAccessibilityLabel
 				accessibilityElements.append(secondaryTitleLabel)
 			}
 
-			if viewModel?.secondaryValue != nil, let secondaryValueLabel = self.secondaryValueLabel {
-				accessibilityElements.append(secondaryValueLabel)
-			}
-
 			if viewModel?.secondaryTrendImage != nil, let secondaryTrendImageView = self.secondaryTrendImageView {
 				accessibilityElements.append(secondaryTrendImageView)
 			}
 
 			if viewModel?.tertiaryTitle != nil, let tertiaryTitleLabel = self.tertiaryTitleLabel {
+				var tertiaryAccessibilityLabel = tertiaryTitleLabel.text
+				if viewModel?.tertiaryValue != nil, let tertiaryValueLabel = self.tertiaryValueLabel {
+					tertiaryAccessibilityLabel?.append(" ")
+					tertiaryAccessibilityLabel?.append(tertiaryValueLabel.text ?? "")
+					tertiaryTitleLabel.accessibilityLabel = tertiaryAccessibilityLabel
+				}
```

Comment:
```suggestion
				}
				tertiaryTitleLabel.accessibilityLabel = tertiaryAccessibilityLabel
```

I guess this also needs to be set in both cases, as you did for the previous two labels